tests/unit/test_rules_visitcount.py

```diff
@@ -20,3 +23,50 @@ def test_visit_count(site, client):
     visit_count = client.session['visit_count']
     assert visit_count[0]['count'] == 2
     assert visit_count[1]['count'] == 1
+
+
+@pytest.mark.django_db
```

Comment:
Is there any reason these tests can't be in a class that inherits from `TestCase`? I learned recently that that's what provides the `@pytest.mark.django_db` mark automatically so it's not needed on every test.

Reply:
I just went with the testing convention that was used for all the other tests. I'm not at all sure why it was done that way.

Reply:
There's some setup done in `fixtures.py` that I don't quite understand and I'm not sure how I'd replicate or call if using TestCase